.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,63 @@
+name: Setup Builds and Tests
+
+on:
+  push:
+    branches: [ master, graphql-migration ]
```

Comment:
Can remove graphql-migration? I will be deleting this branch.

Reply:
:+1: